src/wagtail_personalisation/wagtail_hooks.py

```diff
@@ -173,8 +173,11 @@ def get_context(self):
         context = super(CorrectedPagesSummaryPanel, self).get_context()
 
         pages = utils.exclude_variants(Page.objects.all().specific())
-
-        context['total_pages'] = len(pages) - 1
+        context['total_pages'] = len(pages)
```

Comment:
I don't actually think we should use len() because we do not need to evaluate the queryset.

https://github.com/django/django/blob/5008d59a2a8b9df09737fc05c9e343146a348a90/django/db/models/query.py#L249-L251

We could get away with just using `count()` which should result in a more performant query (probably). May be a good one to change later on.